react-insta-clone/src/components/PostsPage/PostsPage.js

```diff
@@ -0,0 +1,47 @@
+import React, {Component} from 'react';
+import dummyData from '../../dummy-data';
+import SearchBar from '../SearchBar/SearchBar';
+import PostContainer from '../PostContainer/PostContainer';
+
+class PostsPage extends Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+        dummyData : [],
+        searchKeyWord : '',
+        posts: [],
+    }
```

Comment:
If you are going to load up the dummyData to this.state.posts anyway, here you can get rid of dummyData key and just do `posts: dummyData`